Gemfile

```diff
@@ -0,0 +1,22 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.6.3'
+
+gem 'rails', '~> 6.0.0'
+gem 'pg', '>= 0.18', '< 2.0'
+gem 'puma', '~> 3.11'
+gem 'jbuilder', '~> 2.7'
```

Comment:
Interesting. Why do you keep this one around (you mentioned this gem in the PR description, hence my question)

Reply:
Yes, we realised we can use Active Model Serializers instead of jbuilder. Thanks for the heads up!